test/unit/controllers/PeerSelectControllerSpec.js

```diff
@@ -2,22 +2,20 @@
 /* global describe, it, inject, expect, beforeEach */
 
 describe('PeerSelectController', function () {
-  var $controller, $scope, $q, $mod, $APManager, $EService, createController, timeoutTime, $promiseData, $promise, $promiseFlag
-
   beforeEach(module('myApp.controllers'))
 
   beforeEach(function () {
     // The modalInstance will propably usually give a boolean as return.
     // However, for testing purposes it is important to gain knowledge about the input of the function
-    $mod = {
+    this.$modalInstance = {
       close: function (arr) {
         return arr
       }
     }
 
-    timeoutTime = 1000
+    this.waitOneSecond = 1000
```

Comment:
Again, verbs are usually used for functions. `this.oneSecond` should be sufficient?